classification1.Rmd

```diff
@@ -560,17 +646,17 @@ cancer_train <- cancer |>
 cancer_train
 ```
 
-Next, we create a *model specification* for K-nearest neighbours classification
-by calling the `nearest_neighbor` function, specifying that we want to use $K = 5$ neighbours
+Next, we create a *model specification* for $K$-nearest neighbors classification
+by calling the `nearest_neighbor` function, specifying that we want to use $K = 5$ neighbors
 (we will discuss how to choose $K$ in the next chapter) and the straight-line 
 distance (`weight_func = "rectangular"`). The `weight_func` argument controls
-how neighbours vote when classifying a new observation; by setting it to `"rectangular"`,
-each of the $K$ nearest neighbours gets exactly 1 vote as described above. Other choices, 
-which weight each neighbour's  vote differently,  can be found on 
+how neighbors vote when classifying a new observation; by setting it to `"rectangular"`,
+each of the $K$ nearest neighbors gets exactly 1 vote as described above. Other choices, 
+which weigh each neighbor's vote differently, can be found on 
 [the tidymodels website](https://parsnip.tidymodels.org/reference/nearest_neighbor.html).
-We specify the particular computational
-engine (in this case, the `kknn` engine) for training the model with the `set_engine` function.
-Finally we specify that this is a classification problem with the `set_mode` function.
+In the `set_engine` argument, we specify which package or system will be used for training
+the model. In this case, `kknn` is an R package for performing $K$-nearest neighbors classification.
```

Comment:
"In this case, kknn is an R package..." -- sentence a bit awk flow from previous